src/Vault/Cards/Cards.ts

```diff
@@ -0,0 +1,52 @@
+import { Client } from '../../Client'
+import { Resource } from '../../Resource'
+import { DuffelResponse } from '../../types'
+
+export type CardBrand =
+  | 'visa'
+  | 'mastercard'
+  | 'uatp'
+  | 'american_express'
+  | 'diners_club'
+  | 'jcb'
+
+interface CardParameters {
+  address_line_1: string
```

Comment:
If the rest of the code looks good I can add proper docs for these as we do with most other files.